src/apimtemplate/Apim.DevOps.Toolkit.csproj

```diff
@@ -7,7 +7,7 @@
 	<ToolCommandName>dotnet-apim</ToolCommandName>
 	<ProjectGuid>{B5183465-2BC1-4206-9C9F-5AC9615AC941}</ProjectGuid>
 	<PackageId>Apim.DevOps.Toolkit</PackageId>
-	<Version>2.0.0</Version>
+	<Version>2.0.1</Version>
```

Comment:
@andrewdmoreno can you please undo this? Let's keep the version 2.0.0. There are other fixes that need to be added before bumping up the version. Thanks.

Reply:
Removed commit for version change.